### DISCUSSION TITLE
Is mobile phone fingerprint reader supported?

### DISCUSSION THREAD

**Discussion by balki** [authelia/authelia#5489] [Q&A] [answered]
I am trying to replace having to type passwords for some simple PWA webapps with just touch the fingerprint reader in mobile phones. Is this supported? Is this WebAuthn? I found [this](https://www.authelia.com/overview/authentication/security-key/) in docs. But don't plan to use separate hardware device like yubikey.

**Comment by hendrik1120** [comment, marked as answer]
The Platform Authenticator from Webauthn is currently not implemented but is on the roadmap https://www.authelia.com/roadmap/active/webauthn/#platform-authenticator.

**Comment by james-d-elliott** [comment]
Implemented.
